var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -50,6 +57,7 @@ class PyPytest(PythonPackage):
     depends_on('py-attrs@17.2.0:', when='@3.3:3.4', type=('build', 'run'))
     depends_on('py-iniconfig', when='@6.0:', type=('build', 'run'))
     depends_on('py-packaging', when='@4.6:', type=('build', 'run'))
+    depends_on('py-pluggy@0.12:2', when='@6.2.5:', type=('build', 'run'))
```

Comment:
`>=0.12,<2.0` in setuptools is equivalent to `@0.12:1` in Spack (version ranges are inclusive), so you can remove this line